packages/flutter/test/material/text_field_test.dart

```diff
@@ -8405,4 +8405,100 @@ void main() {
     expect(inputWidth, wideWidth);
     expect(cursorRight, inputWidth - kCaretGap);
   });
+
+  group('MaxLengthEnforcement', () {
+    const int maxLength = 5;
+
+    Future<void> setupWidget(
+      WidgetTester tester,
+      TextEditingController controller,
+      MaxLengthEnforcement enforcement,
+    ) async {
+      final Widget widget = MaterialApp(
+        home: Material(
+          child: TextField(
+            controller: controller,
+            maxLength: maxLength,
+            maxLengthEnforcement: enforcement,
+            // TODO(Alex): Remove this param once it's deprecated.
```

Comment:
Use your full Github username here.  That way someone who isn't familiar with this PR can still find you if needed.